openapi-cli/src/main/java/io/ballerina/openapi/cmd/BaseCmd.java

```diff
@@ -62,8 +62,7 @@ public class BaseCmd {
     @CommandLine.Option(names = {"--single-file"}, description = "Generate all contents in a single file")
     public boolean singleFile;
 
-    @CommandLine.Option(names = {"--use-sanitized-oas"}, hidden = true, description = "This is an experimental" +
-            " feature. This option enables code generation by modifying the given OAS to follow the" +
-            " Ballerina language best practices.")
+    @CommandLine.Option(names = {"--use-sanitized-oas"}, hidden = true, description = "This option enables code " +
```

Comment:
Are we going to add the subcommand `bal openapi sanitize` here?

Reply:
No, this PR is to make the existing implementation stable. Introducing `bal openapi sanitize` should be implemented separately on top of these changes